src/components/CopyLink.js

```diff
@@ -81,9 +81,16 @@ export const urlParamsToViewTarget = (url) => {
     result = qs.parse(s[1]);
   }
 
-  // TODO: qs can't tell the difference between false and "false", and "false"
-  // is truthy. So we need to go through and coerce things to real booleans at
-  // some point.
+  // Ensures that the simplify field is a boolean, as the qs module can't tell 
+  // the difference between false and "false" 
+  if (result != null){
+    let resultSimplify = result.simplify;
+    if (resultSimplify === "true"){
+      result.simplify = true;
+    } else if (resultSimplify === "false"){
+      result.simplify = false;
+    }
```

Comment:
I think the `resultSimplify` variable could be eliminated here to improve the code.